tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectWithPrivateFieldsNegativeTest.java

```diff
@@ -68,19 +68,23 @@ public void testPrivateObjAccess1SemanticsNegative() {
         CompileResult compileResult
                 = BCompileUtil.compile("test-src/object/PrivateObjAccess1SemanticsNegativeProject");
         
-        Assert.assertEquals(compileResult.getWarnCount(), 6);
+        Assert.assertEquals(compileResult.getWarnCount(), 10);
         Assert.assertEquals(compileResult.getErrorCount(), 2);
         
         String expectedErrMsg = "attempt to refer to non-accessible symbol ";
-        String expectedWaringMsg = "attempt to expose non-public symbol ";
+        String expectedWarningMsg = "attempt to expose non-public symbol ";
         int i = 0;
         // First error is in a different package
-        BAssertUtil.validateWarning(compileResult, i++, expectedWaringMsg + "'ChildFoo'", 5, 5);
-        BAssertUtil.validateWarning(compileResult, i++, expectedWaringMsg + "'PrivatePerson'", 34, 45);
-        BAssertUtil.validateWarning(compileResult, i++, expectedWaringMsg + "'PrivatePerson'", 38, 1);
-        BAssertUtil.validateWarning(compileResult, i++, expectedWaringMsg + "'PrivatePerson'", 42, 1);
-        BAssertUtil.validateWarning(compileResult, i++, expectedWaringMsg + "'PrivatePerson'", 42, 73);
-        BAssertUtil.validateWarning(compileResult, i++, expectedWaringMsg + "'FooFamily'", 16, 5);
+        BAssertUtil.validateWarning(compileResult, i++, expectedWarningMsg + "'ChildFoo'", 5, 5);
+        BAssertUtil.validateWarning(compileResult, i++, expectedWarningMsg + "'PrivatePerson'", 34, 45);
+        BAssertUtil.validateWarning(compileResult, i++, expectedWarningMsg + "'PrivatePerson'", 38, 1);
+        BAssertUtil.validateWarning(compileResult, i++, expectedWarningMsg + "'PrivatePerson'", 42, 1);
+        BAssertUtil.validateWarning(compileResult, i++, expectedWarningMsg + "'PrivatePerson'", 42, 73);
+        BAssertUtil.validateWarning(compileResult, i++, expectedWarningMsg + "'FooFamily'", 4, 5);
+        BAssertUtil.validateWarning(compileResult, i++, expectedWarningMsg + "'FooFamily'", 6, 5);
+        BAssertUtil.validateWarning(compileResult, i++, expectedWarningMsg + "'FooFamily'", 16, 5);
+        BAssertUtil.validateWarning(compileResult, i++, expectedWarningMsg + "'FooFamily'", 59, 38);
+        BAssertUtil.validateWarning(compileResult, i++, expectedWarningMsg + "'FooFamily'", 66, 44);
```

Comment:
We can add `Assert.assertEquals(compileResult.getWarnCount(), i)` here similar to  https://github.com/ballerina-platform/ballerina-lang/pull/39970/files#diff-4f8066db20149aa8f27874addc97290589b27889ff3d05ff4ee6ba9b2095aa9fL73 so we won't have to change the number of warnings when updating tests.

Reply:
Updated ad671ef6b77b2f6f41f3e3d021ea829778d7819c